src/app/context/patientContext.tsx

```diff
@@ -0,0 +1,27 @@
+import { useContext, createContext } from "react";
+
+export const patientContext = createContext<{ patientData: { [key: string]: string } | null, setPatientData: (patientData: { [key: string]: string }) => void } | null>(null);
+
+export type DataType = {
+    [key: string]: string;
+} | null
```

Comment:
DataType already declared in dataContext and can be imported here, avoid repetition

Reply:
Easier to maintain types specific to certain context, issue with keeping?